apis/core.oam.dev/v1beta1/resourcetracker_types.go

```diff
@@ -207,6 +206,17 @@ func (in *ResourceTracker) AddManagedResource(rsc client.Object, metaOnly bool,
 		OAMObjectReference: common.NewOAMObjectReferenceFromObject(rsc),
 		Deleted:            false,
 	}
+}
+
+// ExistManagedResource check if resource exists in ResourceTracker
+func (in *ResourceTracker) ExistManagedResource(rsc client.Object) bool {
```

Comment:
ExistManagedResource --> IsManagedResourceExisted/CheckManagedResourceExistence?

Reply:
I prefer ExistsManagedResource > DoesManagedResourceExist > CheckManagedResourceExistence. Which one do you think is better?

Reply:
What about ContainsManagedResource?

Reply:
Let's adopt ContainsManagedResource.